nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
Optional. 
You can consider raising the full error if the method is not implemented. Something like this ⬇️
`raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`